pkg/stanza/operator/helper/severity.go

```diff
@@ -14,8 +14,9 @@ import (
 
 // SeverityParser is a helper that parses severity onto an entry.
 type SeverityParser struct {
-	ParseFrom entry.Field
-	Mapping   severityMap
+	ParseFrom     entry.Field
+	Mapping       severityMap
+	overwriteText bool
```

Comment:
Only `overwriteText` is private, is it intentional?

Reply:
Probably the whole struct could be unexported but I only tried to not increase the surface area with this change.